source/simulator/core.cc

```diff
@@ -989,6 +989,8 @@ namespace aspect
   template <int dim>
   void Simulator<dim>::setup_dofs ()
   {
+    signals.edit_parameters(*this, parameters);
+
```

Comment:
Yeah... for now I am putting it back in ```start_timestep()```.  It doesn't matter much from my perspective, and that will cover more potential use cases.